frontend/src/concepts/pipelines/content/pipelinesDetails/pipelineRun/runLogs/LogsTab.tsx

```diff
@@ -322,7 +384,7 @@ const LogsTabForPodName: React.FC<{ podName: string; isFailedPod: boolean }> = (
                         )}
                       </ToolbarItem>
                       <ToolbarItem spacer={{ default: 'spacerNone' }}>
-                        <Dropdown
+                        <DropdownDeprecated
```

Comment:
Why are we using a deprecated component here?

Reply:
@DaoDaoNoCode thanks, updated accordingly